ui/src/app/views/account-creation-success/component.tsx

```diff
@@ -0,0 +1,134 @@
+import {Component, DoCheck, Input, OnInit, ViewChild} from '@angular/core';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {SignInService} from 'app/services/sign-in.service';
+
+import {
+  BolderHeader,
+  Header,
+  SmallHeader
+} from 'app/common/common';
+import {AccountCreationModalsComponent} from 'app/views/account-creation-modals/component';
+import {AccountCreationComponent} from 'app/views/account-creation/component';
+import {LoginComponent} from 'app/views/login/component';
+
+const buttonLinkStyling = {
```

Comment:
should this be within a `styles` block?  is that how we want to define the local styles or are we not doing that anymore?

Reply:
Probably should! Will do.